src/components/sticky/sticky.js

```diff
@@ -43,13 +42,13 @@ export class Sticky extends Component<Props, State> {
 
   render() {
     const { scrollOnTop } = this.state;
-    const { navbarIsBig } = this.props;
     const moving = scrollOnTop ? 'stuck' : ' fixed';
-    const rightPos = navbarIsBig ? 'navbar-is-big' : 'navbar-is-small';
 
     return (
-      <div styleName={`${moving} ${rightPos}`}>
-        {this.props.children}
+      <div id='sticky-root-element'>
+        <div styleName={moving}>
+          {this.props.children}
+        </div>
       </div>
```

Comment:
is the sticky still needed? didn't 'fixed' on right-sidebar adjust this?

Reply:
fixat